lib/foreman_theme_satellite/engine.rb

```diff
@@ -131,7 +131,7 @@ def self.lifecycle_data
   end
 
   def self.get_satellite_version
-    metadata_field('version', '6.13.0-development')
+    metadata_field('version', '6.14.0-development')
```

Comment:
I'm not sure what's best here. Technically I think this should be 6.15.0-development now, since this is ahead of 6.14.0.

Reply:
true, but that will break link checking as there are no 6.15 docs yet.

Reply:
Yes, it's a bit tricky. I'm tempted to use the `index-satellite.html` files on `docs.theforeman.org`, but is that really that useful? The guide names URLs are different, so it's not a great test.

Reply:
Yeah, I'd not do that. The links are totally different and wouldn't be a meaningful test at all.